ui/lib/config-ui/addon/components/messages/page/create-and-edit-message-form.js

```diff
@@ -63,9 +55,9 @@ export default class MessagesList extends Component {
           this.args.message.startTime = new Date(this.args.message.startTime);
         if (typeof this.args.message.endTime === 'string')
           this.args.message.endTime = new Date(this.args.message.endTime);
```

Comment:
same comment here. I understand why, you want to do some conditional logic here. [Here](https://v5.chriskrycho.com/journal/understanding-args-in-glimmer-components/) is a helpful article about why this.args are readOnly.